packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/setup-app.md

```diff
@@ -0,0 +1,7 @@
+4. Select **Web Application** as the **Application type**, then click **Next**.
+5. Specify the **App integration name**.
+6. Select **Authorization Code** as the **Grant type**.
+7. Specify the **Sign-in redirect URIs** to redirect the user with their authorization code.
+8. Fill in the remaining details for your app integration, then click **Save**.
+
+From the **General** tab of your app integration, save the generated **Client ID** and **Client Secret** values to implement your authorization flow.
```

Comment:
why does this not have a number? It can be included as step 8 without confusion.
```suggestion
From the **General** tab of your app integration, save the generated **Client ID** and **Client secret** values to implement your authorization flow.
```